Components/LandingNav.tsx

```diff
@@ -45,51 +48,73 @@ const LandingNav = () => {
           className="flex flex-row items-center cursor-pointer"
           onClick={handleClick}
         >
-           <img src="./logo.svg" alt="logo" className="dark:hidden"/>
-          <img src="./logodark.svg" alt="dark mode logo" className="hidden dark:block"/>
+          <img src="./logo.svg" alt="logo" className="dark:hidden" />
+          <img
+            src="./logodark.svg"
+            alt="dark mode logo"
+            className="hidden dark:block"
+          />
           <h1 className="text-lg font-semibold font-archivo ">GyanaGuru</h1>
         </div>
         {/* Nav Links */}
-        <div className="flex flex-row items-center gap-8">
-          <Link className={navLinkStyle} href="#">
-            Home
-          </Link>
-          <Link className={navLinkStyle} href="#" onClick={handleCoursesClick}>
-            Courses
-          </Link>
-          <Link className={navLinkStyle} href="#ourservices" onClick={handleOurServicesClick}>
-            Our Services
-          </Link>
-          <Link className={navLinkStyle} href="#benefits" onClick={handleBenefitsClick}>
-            Benefits
-          </Link>
-          <Link className={navLinkStyle} href="/faq">
-            {/* adding badge on div for new */}
-            <div>
-              FAQ
-              <span className="absolute px-1 py-1 ml-2 text-xs text-white bg-red-500 rounded-full -top-2">
-                {/* on hover show popup - attention needed */}
-                <div className="w-2 h-2 shadow-xl shadow-red-100"></div>
-
-              </span>
-
-            </div>
-          </Link>
-        </div>
+        {error404Check && (
```

Comment:
when there is no error then show this.
```suggestion
        {!error404Check && (
```

Reply:
done